plugins/uefi/fu-tpm-eventlog-device.c

```diff
@@ -240,31 +264,28 @@ fu_tpm_eventlog_device_parse_blob_v2 (FuTpmEventlogDevice *self,
 						 &datasz, G_LITTLE_ENDIAN, error))
 			return FALSE;
 
-		/* save blob if PCR=0 */
+		/* save blob */
 		idx += sizeof(datasz);
-		if (pcr == ESYS_TR_PCR0) {
```

Comment:
Saving all the bootlog data for every instance is a good chunk of the total RSS, about 60kb from my testing here. I think we only want to collect the PCR0 by default, and parse the rest on demand. I've got an idea for that.

Reply:
Okay